docs/troubleshoot/app-framework/appserver-error-codes.md

```diff
@@ -1800,291 +1800,343 @@ The following error message codes may appear on the app-server log. Use the foll
+### ZWED0070W
+
+  Error when reading PFX. The server cannot continue. Error=_errorMessage_
 
   **Reason:**
 
-  TODO
+  If we get an error while reading _config.https.pfx_ file then the server cannot continue
+  and throws _errorMessage_.
 
   **Action:**
 
-  TODO
+  No action is needed, but the _errorMessage_ may be needed to debug
 
 
 
-### ZWED0071W":"Unexpected error on server %s:%s. E=%s. Stack trace follows.\n%s
+### ZWED0071W
+
+  Unexpected error on server _ipAddress_:_port_. E=_errorMessage_. Stack trace follows. _stack_
 
   **Reason:**
 
-  TODO
+  When we get an unexpected (anything except EACCES, EADDRINUSE, ENOTFOUND, EADDRNOTAVAIL) error
+  in the web server for _ipAddress_:_port_.
 
   **Action:**
 
-  TODO
+  No action needed unless user is experiencing an interruption in server, then send error message and stack to developers
+
 
 
+### ZWED0072W
 
-### ZWED0072W":"Could not stop manager, error=
+  Could not stop manager due to error _errorMessage_
 
   **Reason:**
 
-  TODO
+  If the server manager is unable to stop due to any reason it will throw an exception with an _errorMessage_.
 
   **Action:**
 
-  TODO
+  If the Java manager (handles Jar and War) is unable to stop all servers, send _errorMessage_ to developers
 
 
 
-### ZWED0073W":"No server returned for group=
+### ZWED0073W
+
+  No server returned for group=_group_
 
   **Reason:**
 
-  TODO
+  If No server was found in this War _group_ then it will throw this warning message.
 
   **Action:**
 
-  TODO
+  No action is required
 
 
+### ZWED0074W
 
-### ZWED0074W":"Unknown default behavior=%s
+  Unknown default behavior=_defaultBehavior_
 
   **Reason:**
 
-  TODO
+  The default grouping behaviour in the config for this War is not of type 'microservice' or 'appserver'
 
   **Action:**
 
-  TODO
+  No action is needed, but the warning may be needed to debug
 
 
 
-### ZWED0075W":"Services in plugin=%s war grouping skipped. Plugin missing or already grouped
+### ZWED0075W
+
+  Services in plugin=_plugin_ war grouping skipped. Plugin missing or already grouped
 
   **Reason:**
 
-  TODO
+  Server was not created for _plugin_ War grouping, because it was already made or plugin is missing.
 
   **Action:**
 
-  TODO
+  No action is needed
+
 
 
+### ZWED0076W
 
-### ZWED0076W":"Skipping invalid plugin group=
+  Skipping invalid plugin group=_plugins_
 
   **Reason:**
 
-  TODO
+  If _plugins_ is not an array and the size is less than zero, then it will log a warning message.
 
   **Action:**
 
-  TODO
+  Make sure _plugins_ should be an array of size greater than zero.
+
 
 
+### ZWED0077W
 
-### ZWED0077W":"Could not extract war for service=%s, error=
+  Could not extract war for service=_key-value_, error=_errorMessage_
 
   **Reason:**
 
-  TODO
+  If the service with the _key-value_ pair is unable to extract the war file then it throws the _errorMessage_
 
   **Action:**
 
-  TODO
+  Check if the war file exists and configured correctly.
```

Comment:
Can you update these as per @DivergentEuropeans  suggestions
https://github.com/zowe/docs-site/pull/2526#discussion_r1038317772